pkg/drivers/kic/oci/oci.go

```diff
@@ -84,6 +84,18 @@ func DeleteContainer(ociBin string, name string) error {
 	return nil
 }
 
+// SetupContainerNode sets up the container node
+func SetupContainerNode(p CreateParams) error {
+	if p.OCIBinary != Docker {
```

Comment:
I believe podman preload works as well. ( podman also has docker runtime) so no need to exclude podman.

Reply:
This volume is currently only created for docker:

https://github.com/kubernetes/minikube/blob/d84b9425c9a57f1e8fea7f951229da4c5d0303b4/pkg/drivers/kic/oci/oci.go#L124-L125

is this a bug?

Reply:
I think so... there is no reason we dont create volume for podman.

Reply:
so I was wrong, and you were right. sorry the original code was:

```
	if p.OCIBinary == Podman { // enable execing in /var
		// volume path in minikube home folder to mount to /var
		hostVarVolPath := filepath.Join(localpath.MiniPath(), "machines", p.Name, "var")
		if err := os.MkdirAll(hostVarVolPath, 0711); err != nil {
			return errors.Wrapf(err, "create var dir %s", hostVarVolPath)
		}
		// podman mounts var/lib with no-exec by default  https://github.com/containers/libpod/issues/5103
		runArgs = append(runArgs, "--volume", fmt.Sprintf("%s:/var:exec", hostVarVolPath))
	}
	if p.OCIBinary == Docker {
		if err := createDockerVolume(p.Name, p.Name); err != nil {
			return errors.Wrapf(err, "creating volume for %s container", p.Name)
		}
		glog.Infof("Successfully created a docker volume %s", p.Name)
		runArgs = append(runArgs, "--volume", fmt.Sprintf("%s:/var", p.Name))
		// setting resource limit in privileged mode is only supported by docker
		// podman error: "Error: invalid configuration, cannot set resources with rootless containers not using cgroups v2 unified mode"
		runArgs = append(runArgs, fmt.Sprintf("--cpus=%s", p.CPUs), fmt.Sprintf("--memory=%s", p.Memory))
	}
```